package.json

```diff
@@ -13,6 +13,9 @@
     "@openzeppelin/contracts": "^4.4.2",
     "@solana/web3.js": "^1.29.2",
     "@truffle/hdwallet-provider": "^1.5.0",
+    "@types/config": "^0.0.41",
```

Comment:
IDK why this changes appeared, they should be in develop already as they were added with typescript migration

Reply:
I'm not sure. We have typescript in main dependencies, why should types go to dev dependencies?
Also, I know approach, when docker is not installing dev dependencies, but it's also not our case.

Reply:
Please move typescript to `devDependencies`. `dependencies` should only contain packages that are used in the app's runtime.